Source/WebCore/PlatformGTK.cmake

```diff
@@ -192,3 +192,31 @@ if (USE_GBM)
         ${LIBDRM_LIBRARIES}
     )
 endif ()
+
+if (USE_WHISPER)
+    list(APPEND WebCore_SYSTEM_INCLUDE_DIRECTORIES
+        ${THIRDPARTY_DIR}/whisper.cpp
+    )
+    list(APPEND WebCore_PRIVATE_FRAMEWORK_HEADERS
+        ${THIRDPARTY_DIR}/whisper.cpp/whisper.h
+    )
+    list(APPEND WebCore_SOURCES
+        ${THIRDPARTY_DIR}/whisper.cpp/ggml.c
+        ${THIRDPARTY_DIR}/whisper.cpp/whisper.cpp
+    )
+
+    # These flags suppress compilation errors.
+    set(WHISPER_CPP_COMPILE_FLAGS "-Wno-cast-align -Wno-unused-variable -Wno-error=undef -Wno-error=array-bounds")
```

Comment:
Please don't use `-Wno-error=`. Either suppress the warnings or fix them. Don't make them non-fatal.

Reply:
Then use e.g. `-Wno-undef -Wno-array-bounds` to turn off the warnings entirely. Don't intentionally introduce new warnings! :)

Reply:
Ah, I see. Thanks!